users/views.py

```diff
@@ -249,3 +251,11 @@ def get(cls, request):
             'serializer': users_serializer,
             'users_list': users_queryset,
         })
+
+
+class CustomUserLoginView(LoginView):
+    """
+       Use Django LoginView with diffrent form_class
```

Comment:
Please align the comment. You can also make it one line comment.

Reply:
Sorry for confusion, when I wrote `one line comment` I have meant something like:
```""" Use Django LoginView with diffrent form_class. """```

Class and method comments should be written this way.